cpp/ql/src/experimental/Best Practices/UselessTest.qhelp

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+
+<overview>
+<p>Comparisons operation like <code>a==8 && a!=7</code> contains an useless part : the non-equal part. This rule finds any test of this kind within an if or a while statement
```

Comment:
A couple of grammar fixes :) Also, our PR checks are complaining because of the non-escaped `&&`, so I've replaced it with `&amp;&amp;`.
```suggestion
<p>Comparison operations like <code>a==8 &amp;&amp; a!=7</code> contain a useless part : the non-equal part. This rule finds tests of this kind within an <code>if</code> or a <code>while</code> statement
```